### ISSUE TITLE
How can we make this compatible with fork?

### ISSUE THREAD

**Issue by samskiter** [brechtm/sshgit#6]
Is there a way to take this project further - potentially even to the point it could use the filesystem exposed by VSCode?

I'm unsure how easily we can get 'under' the mmap parts of Fork....

Just brainstorming to bring the best git management tool to VSCode

**Comment by brechtm** [comment]
> Is there a way to take this project further - potentially even to the point it could use the filesystem exposed by VSCode?

Personally, I wouldn't want to limit functionality to VSCode.

> I'm unsure how easily we can get 'under' the mmap parts of Fork....

I'm afraid getting this to work in recent Fork versions is not possible without Dan's support. It is clear that he does not consider this a high-priority feature, which we'll have to respect.

I'm still using Fork 2.31.2 (renamed to RemoteFork as described in the readme) with forkgit on a daily basis. It's not perfect, but it works very well for me.

**Comment by jonasflint** [comment]
I currently tried to implement this using your instructions, but RemoteFork will not launch.  I'm on a M2 mac studio, on Sequoia 15.1 ....

i have force signed the app successfully....

the instructions are not very clear and hard to follow.

**Comment by brechtm** [comment]
> I currently tried to implement this using your instructions, but RemoteFork will not launch.

You can try starting it from the commant line to see whether any errors are printed:
```sh
/Applications/RemoteFork.app/Contents/MacOS/Fork
```

> the instructions are not very clear and hard to follow.

Which instructions are you referring to and what is not clear about them?
